site/specs/numbers.json

```diff
@@ -850,22 +847,24 @@
           "/accounts"
         ],
         "description": "<p>Retrieves the phone numbers according to the input parameters</p>\n",
-        "operationId": "GET /accounts/{accountId}/availableNumbers",
+        "operationId": "GetAvailableNumbers",
+        "summary": "Retrieves available tns",
         "parameters": [
           {
             "name": "accountId",
-            "description": "",
             "in": "path",
             "required": true,
+            "style": "simple",
+            "explode": false,
+            "description": "User's account ID",
+            "example": "900000",
             "schema": {
-              "type": "string",
-              "default": null
-            },
-            "example": null
+              "type": "string"
+            }
           },
           {
             "name": "quantity",
-            "description": "<p>The desired quantity of requested numbers. Values range from 1-5000.</p>\n",
+            "description": "<p>The desired quantity of requested numbers. Values range from 1-5000.</p>",
```

Comment:
We should probably use `minimum` and `maximum` to document the range of values allowed

https://swagger.io/docs/specification/data-models/data-types/#numbers

Reply:
Cool!